### ISSUE TITLE
Yellow focus colour has insufficient contrast against a white background

### ISSUE THREAD

**Issue by paul-request** [Legal-and-General/canopy#552]
The current dandelion yellow focus colour has insufficient focus against a white background.

`--default-focus-color: var(--color-dandelion-yellow);`

**Comment by elenagarrone** [comment]
Just as a note - the design team is looking into this.

**Comment by pauleustice** [comment]
@alexcanning Any progress on this? Is it tracked elsewhere (i.e. can I close this issue down?)

**Comment by alexcanning** [comment]
You can close it, it got fixed.
